compiler/luci/pass/src/ResolveCustomOpAddPass.cpp

```diff
@@ -25,30 +25,49 @@
 namespace
 {
 
+/// @brief Returns the number of Custom("BroadcastTo") node among cop's inputs
+int32_t num_of_broadcast_to(luci::CircleCustom *cop)
```

Comment:
```suggestion
int32_t num_of_broadcast_to(const luci::CircleNode *node)
```
from the functionality of this method, parameter need not to be `CircleCustom`
and would be better `const` type.

Reply:
@seanshpark 

Oh, Thank you for letting me know.
I reflected it. :)